tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_mappings.go

```diff
@@ -170,19 +175,15 @@ func orderFieldMappings(input []dataApiModels.TerraformFieldMappingDefinition) (
 }
 
 func uniqueAndSortModelToModelMappings(input []dataApiModels.TerraformModelToModelMappingDefinition) []dataApiModels.TerraformModelToModelMappingDefinition {
-	keys := make([]string, 0)
 	keysToValues := make(map[string]dataApiModels.TerraformModelToModelMappingDefinition)
 	for _, item := range input {
 		// using this key format means we'll also de-dupe this slice at the same time
 		key := fmt.Sprintf("%s-%s", item.SchemaModelName, item.SdkModelName)
-		keys = append(keys, key)
 		keysToValues[key] = item
 	}
-	sort.Strings(keys)
```

Comment:
This sort is used to ensure the output is consistent, meaning that we don't get spurious changes every regeneration - so this is likely still needed?

Instead we should be able to build up the `keys` map in a separate for loop, to give us a unique, ordered list of keys to map the values across for below

Reply:
Good shout! Done!